src/Microsoft.QualityGuidelines.Analyzers/Core/MarkMembersAsStatic.Fixer.cs

```diff
@@ -22,9 +29,33 @@ public sealed override FixAllProvider GetFixAllProvider()
 
         public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
-            // This is to get rid of warning CS1998, please remove when implementing this analyzer
-            await Task.Run(() => { }).ConfigureAwait(false);
-            throw new NotImplementedException();
+            SyntaxNode root = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+            SyntaxNode node = root.FindNode(context.Span);
+            var model = await context.Document.GetSemanticModelAsync(context.CancellationToken).ConfigureAwait(false);
+            var symbol = model.GetDeclaredSymbol(node, context.CancellationToken);
+
+            context.RegisterCodeFix(new MarkMembersAsStaticAction(
```

Comment:
is symbol guaranteed that symbol being never null? should one can use span from context as it is, or need to search for decl node that can be used in GetDeclaredSymbol?